setup.py

```diff
@@ -35,6 +35,11 @@
 torch_mpi_lib_v2 = Extension('horovod.torch.mpi_lib_v2', [])
 mxnet_mpi_lib = Extension('horovod.mxnet.mpi_lib', [])
 
+have_mlsl = False
+mlsl_root = os.environ.get('MLSL_ROOT')
+if mlsl_root is not None:
```

Comment:
Nit: SImplify to `have_mlsl = mlsl_root is not None`

Reply:
Thank you. Done